src/osd/OSD.cc

```diff
@@ -3551,6 +3559,9 @@ int OSD::run_osd_bench_test(
     }
   }
   utime_t end = ceph_clock_now();
+  dout(0) << __func__
+          << " benchmark took " << end - start
```

Comment:
Can reuse `elapsed` for logging with a precision of 3.

Reply:
done